airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterInput.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling;
+
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
Why is this `@Data` instead of `@Value`? Are we intending to mutate it?

Reply:
We don't but jackson is and this is what cadence uses to serialize the config before storing them in the DB. They need a default ctor which is not possible with `@Value`.